aiomcache/helpers.py

```diff
@@ -0,0 +1,39 @@
+import pickle  # noqa: S403
+from enum import IntEnum
+from typing import Any, Tuple
+
+
+# https://github.com/lericson/pylibmc/blob/master/src/_pylibmcmodule.h#L63
+class PyLibMCFlags(IntEnum):
+    PYLIBMC_FLAG_PICKLE = 1 << 0
+    PYLIBMC_FLAG_INTEGER = 1 << 1
+    PYLIBMC_FLAG_LONG = 1 << 2
+    PYLIBMC_FLAG_BOOL = 1 << 4
+
+
+# see PylibMC_deserialize_native in:
+# https://github.com/lericson/pylibmc/blob/master/src/_pylibmcmodule.c#L640
+async def pylibmc_get_flag_handler(value: Any, flags: int) -> Any:
```

Comment:
Wouldn't the value at this point be bytes?

Reply:
ya i think so, fixed